schemas/content/product-attributes.schema.json

```diff
@@ -0,0 +1,101 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/content/product-attributes",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Product Attributes",
+  "type": "object",
+  "meta:extensible": false,
+  "meta:abstract": false,
```

Comment:
What does this mean, I don't think I have seen it used before in the XDM repo

Reply:
https://github.com/adobe/xdm/blob/master/docs/abstract.md

Reply:
@trieloff is `"meta:abstract"` false by default if omitted?